opensearch-project/OpenSearch-Dashboards
# [Docs] Adds plugin readme's

### Description

This PR adds or updates README files for several plugins in OpenSearch Dashboards, providing more comprehensive documentation. The changes include:

- Updated README for the Embeddable plugin
- Expanded README for the Expressions plugin, including new documentation files
- New README files for the Legacy Export, Newsfeed, and URL Forwarding plugins

These changes aim to improve the developer experience by providing clearer, more detailed documentation for these core plugins.

### Issues Resolved




### Testing the changes

To test these changes:

1. Clone the repository and checkout this branch
2. Navigate to the updated README files in the `src/plugins` directory
3. Verify that the content is readable, well-structured, and provides useful information about each plugin
4. For the Expressions plugin, check that the new documentation files in `src/plugins/expressions/docs/` are accessible and contain relevant information

### Changelog

- doc: Update Embeddable plugin README with more detailed information
- doc: Expand Expressions plugin README and add new documentation files
- doc: Add new README for Legacy Export plugin
- doc: Add new README for Newsfeed plugin
- doc: Update URL Forwarding plugin README with comprehensive information

### Check List
